x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/InferencePyTorchAction.java

```diff
@@ -121,12 +122,19 @@ protected void doRun() throws Exception {
             processor.validateInputs(inputs);
             assert config instanceof NlpConfig;
             NlpConfig nlpConfig = (NlpConfig) config;
+
+            int span = nlpConfig.getTokenization().getSpan();
+            if (chunkResponse && nlpConfig.getTokenization().getSpan() <= 0) {
+                // set to special value that means find and use the default for chunking
+                span = NlpTokenizer.CALC_DEFAULT_SPAN_VALUE;
```

Comment:
Not sure if it isn't feasible but should we be using an enum for these special values instead?

Reply:
Probably not worth the BWC serialisation hassle now but if I was starting from scratch then yes